src/modules/utm/index.ts

```diff
@@ -1,2 +1,2 @@
-export * from './hooks'
-export * from './types'
+export { useInitializeUtm, useUtm } from './hooks'
+export type { UtmParams } from './types'
```

Comment:
Why change the export?
This makes it behave worse IMO as every new export has to manually be added here

Reply:
I think is better to cherry-pick, my hope it is really a reduce number of items the ones exported